src/chef-server-ctl/lib/chef_server_ctl/helpers/key_ctl_helper.rb

```diff
@@ -0,0 +1,192 @@
+require "chef/config"
```

Comment:
This looks like a straight move w/ no changes?

Reply:
Yes; looks like the mv got lost somehow and turned into an add + delete